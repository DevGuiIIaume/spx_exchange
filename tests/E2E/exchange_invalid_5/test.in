2
[T0] SELL 0 GPU 10 500 ;
[T1] BUY   0   Router   50   150;
[T1] BUY 0 Router 50 150;
[T1] AMEND  0 51 151;
[T1] CANCEL 0 ;
[T1] CANCEL 0;
[T0] DISCONNECT;
[T1] DISCONNECT;