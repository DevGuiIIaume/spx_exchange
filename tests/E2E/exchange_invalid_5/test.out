[SPX] Starting
[SPX] Trading 2 products: GPU Router
[SPX] Created FIFO /tmp/spx_exchange_0
[SPX] Created FIFO /tmp/spx_trader_0
[SPX] Connected to /tmp/spx_exchange_0
[SPX] Connected to /tmp/spx_trader_0
[SPX] Created FIFO /tmp/spx_exchange_1
[SPX] Created FIFO /tmp/spx_trader_1
[SPX] Connected to /tmp/spx_exchange_1
[SPX] Connected to /tmp/spx_trader_1
[SPX] [T0] Parsing command: <SELL 0 GPU 10 500 >
[SPX] [T1] Parsing command: <BUY   0   Router   50   150>
[SPX] [T1] Parsing command: <BUY 0 Router 50 150>
[SPX]	--ORDERBOOK--
[SPX]	Product: GPU; Buy levels: 0; Sell levels: 0
[SPX]	Product: Router; Buy levels: 1; Sell levels: 0
[SPX]		BUY 50 @ $150 (1 order)
[SPX]	--POSITIONS--
[SPX]	Trader 0: GPU 0 ($0), Router 0 ($0)
[SPX]	Trader 1: GPU 0 ($0), Router 0 ($0)
[SPX] [T1] Parsing command: <AMEND  0 51 151>
[SPX] [T1] Parsing command: <CANCEL 0 >
[SPX] [T1] Parsing command: <CANCEL 0>
[SPX]	--ORDERBOOK--
[SPX]	Product: GPU; Buy levels: 0; Sell levels: 0
[SPX]	Product: Router; Buy levels: 0; Sell levels: 0
[SPX]	--POSITIONS--
[SPX]	Trader 0: GPU 0 ($0), Router 0 ($0)
[SPX]	Trader 1: GPU 0 ($0), Router 0 ($0)
[SPX] Trader 0 disconnected
[SPX] Trader 1 disconnected
[SPX] Trading completed
[SPX] Exchange fees collected: $0
